test/collection_test.js

```diff
@@ -827,9 +827,7 @@ describe("Collection", () => {
       it("should resolve with imported deletions", () => {
         return articles.pullChanges(result)
           .then(res => res.deleted)
-          .should.eventually.become([
-            {id: id_4}
-          ]);
+          .should.eventually.become([id_4]);
```

Comment:
I debated that; yes it's a breaking change but couldn't see the necessity of keying the id. Can definitely revert to the old behavior.